openstack/networking/v2/ports/requests.go

```diff
@@ -168,12 +168,13 @@ type UpdateOptsBuilder interface {
 
 // UpdateOpts represents the attributes used when updating an existing port.
 type UpdateOpts struct {
-	Name           string
-	AdminStateUp   *bool
-	FixedIPs       interface{}
-	DeviceID       string
-	DeviceOwner    string
-	SecurityGroups []string
+	Name                string
+	AdminStateUp        *bool
+	FixedIPs            interface{}
+	DeviceID            string
+	DeviceOwner         string
+	SecurityGroups      []string
+	AllowedAddressPairs interface{}
```

Comment:
I'm not sure why we used an `interface{}` for the `FixedIPs`. We may not have found documentation for them when that was written (i.e. is it an array of objects, an array of strings, something else, etc). Regardless, let's use `[]AddressPair` if we know that's the format. Same thing on line 107.